.gitignore

```diff
@@ -5,3 +5,4 @@
 /vendor/
 /zf-mkdoc-theme.tgz
 /zf-mkdoc-theme/
+/.idea
```

Comment:
Please don't mind this change, it will be reverted when I complete the PR

Reply:
:+1: